sync.py

```diff
@@ -172,8 +174,9 @@ def sync_metarepo(clone_dir, name, submodules):
 
     for submodule in submodules_missing:
         logging.debug('Adding submodule %s to meta repository %s', submodule, name)
-        args = shlex.split('git submodule add -b master git@github.com:{}/{}.git'.format(ORGANIZATION, submodule))
-        subprocess.check_call(args, cwd=metarepo_dir)
+        subprocess.check_call(
+            shlex.split('git submodule add -b master git@github.com:{}/{}.git'.format(ORGANIZATION, submodule)),
```

Comment:
Can we use `args = ...` as above?

Reply:
I'm actually trying to eliminate `args` because
1. the variable isn't used again with that same value
2. the naming is confusing
3. some places don't use it, some do

Reply:
Fair enough. What do you think about `add_args`, `update_args`, etc?

Reply:
For the reasons above I think the variable isn't needed at all. My latest commit consistently eliminates all uses in such contexts.